guardian/recipes-backend
# Update CDK and dependencies

## What does this change?

Bumps CDK version and associated libraries via `npm audit fix`

## How to test

- Build and deploy to CODE
- Author a recipe out to CODE and check it's in the backend. If it works, all good!
